App.js

```diff
@@ -150,6 +159,26 @@ class App extends React.Component<Props, *> {
     executeDeepLink(deepLink);
   };
 
+  showDarkModeAlert = () => {
+    const { markThemeAlertAsShown, changeAppTheme, seenThemeAlert } = this.props;
+    if (seenThemeAlert) return;
+    const defaultPreference = Appearance.getColorScheme();
+    if (defaultPreference === DARK_PREFERENCE) {
+      markThemeAlertAsShown();
+      Alert.alert(
+        'Dark mode available',
+        'Would you like to turn on Dark mode now? You can always switch between modes in settings.',
+        [
+          { text: 'Cancel', onPress: () => markThemeAlertAsShown() },
+          {
+            text: 'Turn on',
+            onPress: () => changeAppTheme(DARK_THEME),
+          },
+        ],
+      );
+    }
```

Comment:
I know these kind of comments are annoying, but maybe just `if (defaultPreference !== DARK_PREFERENCE) return`? 😄

Reply:
I'd say it's ok without it on functions with less than 20 lines of code